pkg/codegen/internal/test/testdata/regress-node-8110/go/my8110/pulumiTypes.go

```diff
@@ -0,0 +1,64 @@
+// *** WARNING: this file was generated by test. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
```

Comment:
Non-node code noisy, relatively low value, we should probably support skipping codegen by lang.